bin/new_plugin_repo.rb

```diff
@@ -10,7 +10,7 @@
   ManageIQ::Release.common_options(self, :except => :repo_set)
 end
 
-repo = ManageIQ::Release.repo_for(opts[:repo])
```

Comment:
what is the rationale for this change?  seems unrelated to the PR title.

Reply:
When I add new repos I run the bin/new_plugin_repo.rb, and these couple of changes were bugs found in that effort.